db/migrate/20211230202951_create_last_wrangling_activities.rb

```diff
@@ -0,0 +1,14 @@
+class CreateLastWranglingActivities < ActiveRecord::Migration[5.2]
+  def change
+    create_table :last_wrangling_activities do |t|
+      t.references :user,
+                   foreign_key: true,
```

Comment:
I think [the current plan is to avoid foreign keys](https://otwarchive.atlassian.net/browse/AO3-5690?focusedCommentId=358649), since they don't play nicely with online schema migrations on large tables.